.prettierrc.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
nit: here and throughout
```suggestion
 * Copyright 2021 Google LLC
```